alex/components/dm/base.py

```diff
@@ -214,14 +208,3 @@ def log_state(self):
         :return: none
         """
         self.dialogue_state.log_state()
-
-    def get_token(self):
```

Comment:
OK. I moved it to utils/token.py because I think this is a better place for such method as it is not a general aspect of a dialog manager.